openmls/src/binary_tree/array_representation/diff.rs

```diff
@@ -277,7 +281,10 @@ impl<'a, T: Clone + Debug> AbDiff<'a, T> {
             .iter()
             .position(|&direct_path_node_index| direct_path_node_index == subtree_root_node_index)
             // The shared subtree root has to be in the direct path of both nodes.
-            .ok_or(ABinaryTreeDiffError::LibraryError)
+            .ok_or_else(|| {
+                LibraryError::custom("ABDiff::node_by_index(): index should be in the direct path")
```

Comment:
Putting function names in comments is always dangerous as they might go out of date.